tools.md

```diff
@@ -10,3 +10,9 @@ A collection of useful tools
   * [Drop all tables within active schema](https://gist.githubusercontent.com/dextermb/3811bb599286a8ee558efc10c2f57419/raw/70d5a86154f6d96872f279060fe14e065e2f3785/drop_all_tables.sql)
   * [Remove remoteless (dead) branches](https://gist.githubusercontent.com/dextermb/04c40d02a8e73fc52c90578824ca62ff/raw/201dd731d9c4d5ee6783e965d57578908fe1b467/delete_remoteless_branches.sh)
   * [Command typo recovery](https://github.com/nvbn/thefuck)
+
+# Files
+
+A collection of useful files
+
+* [.gitignore](gitignore.md)
```

Comment:
`gitignore.md` does not appear to exist?

Reply:
As it's a relative link it goes to the PR. But viewing the branch and clicking the link works

Reply:
I tried this before I commented. Clicking the link from the branch here does not work. https://github.com/langleyfoxall/technologies/blob/feature/update-gitignore/tools.md

Reply:
🅾️ I see why haha

Reply:
@DivineOmega Too many markdown files 😄